Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -584,7 +584,7 @@ static inline void GeneratePixelShader(T& out, DSTALPHA_MODE dstAlphaMode, API_T
 	{
 		out.Write("\tint zCoord = int(rawpos.z * 16777216.0);\n");
 	}
-	out.Write("\tzCoord = clamp(zCoord, " I_ZBIAS"[1].x - " I_ZBIAS"[1].y, " I_ZBIAS"[1].x);\n");
+	out.Write("\tzCoord = clamp(zCoord, 0, 0xFFFFFF);\n");
```

Comment:
I think we need a hardware test here to check which of this clamping is correct :/